src/platform_impl/android/mod.rs

```diff
@@ -43,6 +44,10 @@ fn poll(poll: Poll) -> Option<EventSource> {
 
 pub struct EventLoop<T: 'static> {
     window_target: event_loop::EventLoopWindowTarget<T>,
+    // This read guard will be held between each pair of `Resumed` and `Suspended`
+    // events to ensure that `ndk_glue` does not release the `NativeWindow` prematurely,
+    // invalidating raw handles obtained by the user through `Window::raw_window_handle()`
+    native_window_lock: Option<RwLockReadGuard<'static, Option<NativeWindow>>>,
```

Comment:
You should use triple slashes (`///`) here to turn this into a documentation comment, like the suggestion described. That way it shows up in `rustdoc` output and editors/IDEs with `rustdoc` support like `rust-analyzer` :grimacing: !

Reply:
More specifically you can (and should!) convert all item references to intra-doc-links:

```suggestion
    /// This read guard will be held between each pair of
    /// [`event::Event::Resumed`] and [`event::Event::Suspended`] events to
    /// ensure that [`ndk_glue`] does not release the [`NativeWindow`]
    /// prematurely, invalidating raw handles obtained by the user through
    /// [`Window::raw_window_handle()`]
    native_window_lock: Option<RwLockReadGuard<'static, Option<NativeWindow>>>,
```

If you find the full path too long you can also use `` [`Resumed`](event::Event::Resumed) ``.